src/app/components/simple-selectable-list/SimpleSelectableListItem.jsx

```diff
@@ -42,11 +44,28 @@ export default class SimpleSelectableListItem extends Component {
 
     render() {
         const details = this.props.details || [];
+        const extraDetails = this.props.extraDetails || [];
+        const colCount = this.props.colCount;
         return (
             <li className="simple-select-list__item">
-                {this.renderTitle()}
-                {details.map((detail, i) => {
-                    return <p key={`detail-${i}`}>{detail}</p>;
+                <div className={`simple-select-list__col simple-select-list__cols-${colCount}`}>
+                    {this.renderTitle()}
+                    {details.map((detail, i) => {
+                        return <p key={`detail-${i}`}>{detail}</p>;
+                    })}
+                </div>
+                {extraDetails.map((column, i) => {
+                    return (
+                        <div key={`detail-${i}`} className={`simple-select-list__col simple-select-list__cols-${colCount}`}>
+                            {column.map((detail, j) => {
+                                return (
+                                    <span key={`detail-${j}`} className={detail.classes}>
+                                        {detail.content}
```

Comment:
The reason I went for a 2D array mapping to individual spans was to be able to support things like the 'chip' icons where it felt like they needed to individual elements rather than a combined string.